src/services/AnalysisArgumentParser.ts

```diff
@@ -6,71 +6,65 @@ import {
   LogLevel,
   OnFailure,
   ScanType,
+  ScmType,
 } from "../enums";
 import { SOOS_CONSTANTS } from "../constants";
 import { ensureEnumValue, ensureNonEmptyValue, getEnvVariable } from "../utilities";
+import { SOOS_CONTRIBUTOR_AUDIT_CONSTANTS } from "./ContributingDeveloperAuditService/constants";
 
-const getIntegrateUrl = (scanType: ScanType) =>
-  `${SOOS_CONSTANTS.Urls.App.Home}integrate/${
-    scanType == ScanType.CSA ? "containers" : scanType.toLowerCase()
-  }`;
+const getIntegrateUrl = (scanType?: ScanType): string => {
+  const baseMessage = `${SOOS_CONSTANTS.Urls.App.Home}integrate/`;
+  if (!scanType) return `${baseMessage}${ScanType.SCA.toLowerCase()}`;
 
-interface IBaseScanArguments {
-  apiKey: string;
-  apiURL: string;
+  return `${baseMessage}${scanType == ScanType.CSA ? "containers" : scanType.toLowerCase()}`;
```

Comment:
I think you can just fall back here instead of the early return
```suggestion
  return `${baseMessage}${scanType == ScanType.CSA ? "containers" : (scanType ? scanType :  ScanType.Sca).toLowerCase()}`;
```